rpgpy/spcutil.py

```diff
@@ -129,3 +129,46 @@ def find_peak_edges(signal: np.array) -> Tuple[int, int]:
 
     return edge_left, edge_right
 
+
+def spectral_LDR_Galetti(spec_HV: np.array, spec_H: np.array, spec_VH_Re:np.array, spec_VH_Im: np.array,
+                         noise_HV: np.array, noise_H: np.array, num_FFT: np.array, num_rangebins: int,
+                         range_offsets: np.array, version: int) -> np.array:
+    """
+    Compute spectral (S)LDR for vertically pointing radar according to the method by Galetti et al. (2012);
+    Based on code by Alexander Myagkov (RPG).
+    For RPG radars software version > 5.40, the combined spectrum is normalized by 4 (previously 2)
+    Args:
+        spec_HV: H+V channel reflectivity spectra
+        spec_H: only horizontal channel power spectra
+        spec_VH_Re: covariance spectrum Re
+        spec_VH_Im: covariance spectrum imaginary part
+        noise_V: noise power V channel
+        noise_HV: noise power H+V channel
+        num_FFT: number of FFT points
+        num_rangebins: number of range layers
+        range_offsets: range offsets
+        version: software version of RPG radar
+
+    Returns:
+        np.array: SLDR array in dB units
+
+    """
+    if version < 5.40:
```

Comment:
Version is actually 540 (i.e. version * 100) in the header. It need to be divided by 100 at some point or this line fixed. Either way is fine for me.

Reply:
thanks for spotting this. I changed it to 540.